### DISCUSSION TITLE
Move feature request to GitHub

### DISCUSSION THREAD

**Discussion by Josh65-2201** [obsproject/obs-studio#5177] [General]
https://ideas.obsproject.com This extra site and account that has to be made and another area I have to look through is not coinvent and GitHub editing/management is a lot better in my opinion. Feature requests should be moved to here or to the issues section with the enchantment tag.
The site should be set to read only and have a notice to point future issues to discussions or issues page. The link should also be changed in issues templates.

**Comment by Fenrirthviti** [comment]
GitHub would also require an account to be made, so it does not solve the concern you have here. We are not considering moving at this time.

**Reply by Josh65-2201** [reply]
I know an account has to be made here but for user who don't use GitHub they make 1 account just like if they were to on the feedback site. But for people who do have a GitHub account can manage all the issues and feedback in one place and don't have to make an extra account for feedback only.

**Reply by derrod** [reply]
Creating a GitHub account is already a higher barrier to entry and teaching users how to navigate github for a simple feature request is not reasonable.

That aside, you can just log in with github on our Fider anyway.
